live-examples/js-examples/string/string-substring.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">const str = 'Mozilla';
+console.log(str.substring(1,2)); // 'oz'
+console.log(str.substring(2,str.length)); 'zilla'
```

Comment:
There's no point doing this: `str.substring(2,str.length)`, because it's just the same as `str.substring(2)`.

In fact let's use `str.substring(2)` instead on this line, then we can illustrate the optional parameter.